routers/imagesRouter.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+const express = require('express')
+const {authroizationMiddlware} = require('@first-lego-league/ms-auth')
+
+const {getAllImages, getImage, saveImageFromBase64, deleteImage} = require('../logic/imagesLogic')
+
+const adminAction = authroizationMiddlware(['admin', 'development'])
+
+const router = express.Router()
+
+router.get('/all', (req, res) => {
+  getAllImages().then(images => {
+    res.send(images)
+  }).catch(err => {
+    res.status(500)
+    res.send(err.message)
+  })
+})
+
+router.get('/:imageName', (req, res) => {
+  getImage(req.params.imageName).then(image => {
+    res.send(image)
+  }).catch(err => {
+    res.status(500)
+    res.send(err.message)
+  })
+})
+
+router.post('/', adminAction, (req, res) => {
+  saveImageFromBase64(req.body.name, req.body.image).then(() => {
```

Comment:
The client has to send you a base64 of the image? You already have a package to do this yourself, why can't the cient simply give you a file?

Reply:
I think its easier to send text, what do you think?

Reply:
From the client? I dont see the difference between sending a file and a text, as long as it's not part of a larger form, which it isn't. The cilent shouldn't need to worry about the base64 operations if it doesn't have to

Reply:
The difference is the parsing on the server side. If we define that we get it as a text, I dont need special handling of form data and i can get it as json

Reply:
This should not be a worry for the client. The client is now bound to the Base64 implementation that the server chose. The image has to be a file, but ot doesn't have to be in Base64. You should stick to what's constant.

About the "special handling":
This is [formidable](https://www.npmjs.com/package/formidable), which does the work for you, and here is a [W3Schools guid](https://www.w3schools.com/nodejs/nodejs_uploadfiles.asp) on how to use it.